src/Search/Signals.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace eLife\Search;
+
+class Signals
+{
+    private static $isValid = true;
+
+    public static function register()
+    {
+        if (function_exists('pcntl_signal')) {
```

Comment:
when is this not present? I can think of Windows for example, but I'd like a clear error message rather than a no-op as it's important functionality